crates/factor-outbound-http/src/wasi.rs

```diff
@@ -114,13 +164,198 @@ impl WasiHttpView for WasiHttpImplInner<'_> {
                     self.state.request_interceptor.clone(),
                     self.state.self_request_origin.clone(),
                     self.state.blocked_networks.clone(),
+                    http_clients,
```

Comment:
This is mostly out of scope so feel free to ignore but it seems clear that these `state` fields should clearly be their own struct with `send_request` as a method. I have no idea why I didn't do that last time I was in here...